poc/frost.sage

```diff
@@ -235,17 +247,17 @@ for (name, G, H) in ciphersuites:
     group_comm = signers[1].group_commitment(commitment_list, binding_factor)
 
     round_one_outputs = {
-        "participants": [str(index) for index in participant_list],
+        "participants": ",".join([str(index) for index in participant_list]),
         "commitment_list": to_hex(rho_input),
```

Comment:
I think this needs a better name as it's different than just the public commitments being used

Reply:
What is "this"? Are you referring to `commitment_list` or `participants`?

Reply:
Sorry, `commitment_list`

Reply:
True -- it's the encoded group commitment list plus the message hash. I'll change the name to `binding_factor_input`.